app/views/content_items/travel_advice.html+print.erb

```diff
@@ -0,0 +1,34 @@
+<%
+  content_for :title, "Print #{@content_item.page_title}"
+  content_for :simple_header, true
+  content_for :extra_head_content do %>
+  <meta name="robots" content="noindex, nofollow">
+  <script>window.onload = function() { window.print(); }</script>
```

Comment:
Seems like overkill. I'd expect to use a nonce, etc. Can fix when we use CSP. (This is a direct port)